packages/test-utils/src/wrapper-array.js

```diff
@@ -70,25 +62,25 @@ export default class WrapperArray implements BaseWrapper {
   hasAttribute (attribute: string, value: string): boolean {
     this.throwErrorIfWrappersIsEmpty('hasAttribute')
 
-    return this.wrappers.every(wrapper => wrapper.hasAttribute(attribute, value))
+    return this.every(wrapper => wrapper.hasAttribute(attribute, value))
   }
 
   hasClass (className: string): boolean {
     this.throwErrorIfWrappersIsEmpty('hasClass')
 
-    return this.wrappers.every(wrapper => wrapper.hasClass(className))
+    return this.every(wrapper => wrapper.hasClass(className))
```

Comment:
Not part of PR, but I think that developers may be a confused => when use `hasClass` from `WrapperArray` someone may expect that **any** of children has specified class:

    this.some(wrapper => wrapper.hasClass(className))